bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/DPIZoomChangeRegistry.java

```diff
@@ -48,13 +49,17 @@ public static void applyChange(Widget widget, int newZoom, float scalingFactor)
 		if (widget == null) {
 			return;
 		}
+		int initialHandlerCount = dpiZoomChangeHandlers.entrySet().size();
```

Comment:
No need to risk instantiating a new set:
```suggestion
		int initialHandlerCount = dpiZoomChangeHandlers.size();
```